project/JdkOptions.scala

```diff
@@ -37,7 +37,7 @@ object JdkOptions extends AutoPlugin {
       targetSystemJdk,
       jdk8home,
       fullJavaHomes,
-      Seq("-target:jvm-1.8"),
+      Seq(if (Dependencies.getScalaVersion().startsWith("3.0")) "-Xtarget:8" else "-target:jvm-1.8"),
```

Comment:
Ok :)

Reply:
This is for the `jdk8options` parameter; I think we could also just pass `Nil` here; when building on 8, `-target:8` is the default. What do you think?

Reply:
IIRC `jdk8options` is for builds that should target jdk8 (while we build on jdk11+), so we probably need it. I was just surprised the option changed, but it's :+1: .

(yes, we'd ideally use `-release` instead, but #27079)

Reply:
In `private def selectOptions` there's `... else if (isJdk8) jdk8options else ...`; it seems the next parameter is for settings when building on 11.

Reply:
ah, I misremembered. OK anyhow :)